Create Page for 62nd DPP Mini-Conf Follow-up Meetings (#78)
PlasmaPy/plasmapy.github.io: web/pages/meetings/aps/62nd_dpp_mini_conf_followups.md

title: 62nd APS DPP - 11.04 Mini-Conference Followups
author: Erik T. Everson
hidetitle: True

[Zoom link]: https://harvard.zoom.us/j/91600794594?pwd=L09iTGtTRUN1RmpsVnNvU05LRnNwQT09
[chat room]: https://app.element.io/#/room/#plasmapy:openastronomy.org

# 62nd APS DPP - 11.04 Mini-Conference | Post-Meeting Discussions
### Growing an open-source software ecosystem for plasma science

<div style="height: 12px"><!-- Adding vertical whitespace --></div>

> **Where:** On Zoom [link will be updated] <br/>

Reviewer: ```suggestion
**Where:** [Zoom link] <br/>
```
We could add the Zoom link in now.  (Hopefully I'm remembering how to use Markdown links right...)

**Day 1:** Tuesday, November 17th at 21:00 UTC / 16:00 ET / 13:00 PT <br/>
**Day 2:** Thursday, November 19th at 19:00 UTC / 14:00 ET / 11:00 PT <br/>

<div style="height: 12px"><!-- Adding vertical whitespace --></div>

> In order to capitalize on the incredible information shared in this year's

Reviewer 2: In order to capitalize on the opportunities afforded by bringing together the diversity of speakers and attendees during this year's

> mini-conference, we plan to host a couple **post-meeting discussions**.  Here

Reviewer: ```suggestion
mini-conference, we plan to host two **post-meeting discussions**.  Here
```

> In order to capitalize on the incredible information shared in this year's
> mini-conference, we plan to host a couple **post-meeting discussions**.  Here
> we can continue the discussion on fostering the open-source community, bringing
> in more members to our projects, organizing inter-project efforts, improving
> package interoperability, etc.

Reviewer: ```suggestion
In order to capitalize on the incredible information shared in this year's
mini-conference, we plan to host two **post-meeting discussions**.  Here
we can continue the discussion on fostering the open-source community, bringing
in more members to our projects, organizing inter-project efforts, improving
package interoperability, etc.
```
